source/js/openstreetmap/addMarkers.ts

```diff
@@ -0,0 +1,144 @@
+import { getMarkerDataFromElement, pushCoordinatesToBrowserHistory } from './helpers/osmHelpers';
+import L, { Map as LeafletMap, Marker, MarkerClusterGroup } from 'leaflet';
+import { Location, Tooltip, Icon } from './interface/interface';
+
+class AddMarkers {
+    map?: LeafletMap;
+    markers?: MarkerClusterGroup;
+    container?: HTMLElement;
```

Comment:
We should probably do these null checks before instantiation.
```suggestion
    map: LeafletMap;
    markers: MarkerClusterGroup;
    container: HTMLElement;
```

Reply:
I removed the ?, since it is checked in the main file before the class is instantiated